### PR TITLE
Manage reSolve Applications

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: docs/manage-application.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+id: manage-application
+title: Manage a ReSolve Application
+---
+
+Use the [resolve-module-admin](https://github.com/reimagined/resolve/tree/master/packages/modules/resolve-module-admin#readme) CLI tool to manage a reSolve application's sagas and read models. It provides the following functionality:
```
Comment (albertov05) [inline docs/manage-application.md:6]: `It provides the following functionality:`

I'd like to update this without using "provides" if possible. Can we say "includes" instead?

File: docs/manage-application.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- List application read models and sagas.
+- Pause and resume updates for read models and sagas.
+- Reset read model and saga persistent state.
+- Manage saga properties.
+
+You can use **npx** to run this tool without installation or install it as an application's dependency. See the [Installation](#installation) section for more information.
+
+## Installation
+
+Use the steps bellow to add the **resolve-module-admin** tool to a reSolve application.
```
Comment (albertov05) [inline docs/manage-application.md:17]:
```suggestion
Use the steps below to add the **resolve-module-admin** tool to a reSolve application.
```

File: docs/manage-application.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Usage
+
+The **resolve-module-admin** tool communicates with API handlers exposed by a reSolve application. To use the tool, you need to start your application. Then, you can execute **resolve-module-admin** with the required command:
+
+```bash
+yarn resolve-module-admin read-models reset ShoppingList
+```
+
+The default URL that **resolve-module-admin** uses to access a reSolve application's API is `http://localhost:3000/api`. Use the `--api-url` option to specify other URL:
```
Comment (albertov05) [inline docs/manage-application.md:43]:
```suggestion
The default URL that **resolve-module-admin** uses to access a reSolve application's API is `http://localhost:3000/api`. Use the `--api-url` option to specify another URL:
```

File: docs/manage-application.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+```bash
+yarn resolve-module-admin --api-url "https://127.0.0.1:2000/api" read-models reset ShoppingList
+```
+
+For the full list of supported commands, refer to the [NPM package's description](https://github.com/reimagined/resolve/tree/master/packages/modules/resolve-module-admin#readme) or use the inbuilt help:
```
Comment (albertov05) [inline docs/manage-application.md:49]: What is "inbuilt help"? Do you mean "built-in help"?
Also, can we specify what "help" is? For example, if it's a feature or command, can we say "built-in help feature" or "built-in help command"?

File: docs/manage-application.md (hunk 1 above)
Comment (EugeniyBurmistrov) [reply docs/manage-application.md]: Yes, changed to "includes"

File: docs/manage-application.md (hunk 4 above)
Comment (EugeniyBurmistrov) [reply docs/manage-application.md]: Yes, thank you. I changed it to "built-in help command".

File: docs/manage-application.md (hunk 1 above)
Comment (albertov05) [reply docs/manage-application.md]: Thank you.

File: docs/manage-application.md (hunk 4 above)
Comment (albertov05) [reply docs/manage-application.md]: Thank you!

File: docs/manage-application.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+   ```bash
+   yarn add resolve-module-admin
+   ```
+
+2. Register a script that runs the **resolve-module-admin** tool in the application's **package.json** file:
+
+   ```js
+   "scripts": {
+     ...
+     "resolve-module-admin": "resolve-module-admin",
```
Comment (const314) [inline docs/manage-application.md:28]: Maybe it's better to change the script to 'admin' or 'manage' or something similar for convenience.
Command will look better in this case, for example: 
yarn manage read-models reset ShoppingList

**Review by const314 — APPROVED** [review]

**Pull request merged**